### PR TITLE
[r19.03] ghostscript: catch up with many CVE patches

### PR DESCRIPTION
**Pull request by risicle**
###### Motivation for this change
Having the same base version as 19.09, it's easy enough to bring all the relevant patches up to the level of #73590.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/misc/ghostscript/default.nix [hunk 1]
```diff
@@ -51,6 +51,37 @@ stdenv.mkDerivation rec {
       url = "http://git.ghostscript.com/?p=ghostpdl.git;a=patch;h=d3537a54740d78c5895ec83694a07b3e4f616f61";
       sha256 = "1hr8bpi87bbg1kvv28kflmfh1dhzxw66p9q0ddvbrj72qd86p3kx";
     })
+    ./9.26-CVE-2019-3835-part-1.patch
```
Comment (doronbehar) [inline pkgs/misc/ghostscript/default.nix]: So why did you use `fetchpatch` for some patches and local files for other patches?

Comment (risicle) [reply pkgs/misc/ghostscript/default.nix]: I think a couple of them needed manual adaptation. But this was a long time ago.

**Comment by doronbehar** [comment]
@risicle this PR targets an unmaintained branch right? Perhaps we should close it now?

**Comment by risicle** [comment]
If ya like, I don't see any harm in throwing a bone to people stuck on an old release tho :shrug:

**Pull request closed without merging**